database/seeds/DatabaseSeeder.php

```diff
@@ -31,6 +31,5 @@ public function run()
         $this->call('StatusPageHelpTextTableSeeder');
         $this->call('TrackingCodeSettingTableSeeder');
         $this->call('UsersTableSeeder');
-        $this->call('WaterGateSeeder');
```

Comment:
What was this for?!? haha

Reply:
From what I could find, seemed like some kind of situation where there were accidentally two databases? https://github.com/DoSomething/longshot/pull/618